src/Common/src/Interop/Windows/WinSock/Interop.WSAConnect.cs

```diff
@@ -10,10 +10,9 @@ internal static partial class Interop
 {
     internal static partial class Winsock
     {
-        // This function is always potentially blocking so it uses an IntPtr.
         [DllImport(Interop.Libraries.Ws2_32, SetLastError = true)]
         internal static extern SocketError WSAConnect(
-            [In] IntPtr socketHandle,
+            SafeSocketHandle socketHandle,
```

Comment:
I've removed one more on Windows with the `UpdateAcceptContext` socket option. The handle was used in a special way, I'm not entirely sure if it was needed.
The remaining ones are in the `Socket.Select` implementations. Since those involve lists of file descriptors, perhaps I can tackle those in a different PR.